Mathlib/Algebra/Group/Action/Defs.lean

```diff
@@ -71,14 +71,22 @@ lemma op_smul_eq_mul {α : Type*} [Mul α] (a b : α) : MulOpposite.op a • b =
 @[to_additive (attr := simp)]
 lemma MulOpposite.smul_eq_mul_unop [Mul α] (a : αᵐᵒᵖ) (b : α) : a • b = b * a.unop := rfl
 
-/-- Type class for additive monoid actions. -/
+/-- Type class for additive monoid actions.
+
+For example, if `A` is an additive group and `X` is a type, the way to
+say "let `A` act on `X`" is `[AddAction A X]`.
+-/
```

Comment:
Or maybe just
```suggestion
/--  The `AddAction G P` typeclass says that `G` acts additively on `P`. -/
```

Reply:
As a mathematician I know *exactly* what "let the group G act on the set S" means. But I don't really know what it means for an additive monoid to act additively, I find this sentence much more confusing (and indeed 8 years ago I didn't even know what a monoid was). What I want to put into the docstrings is examples of situations which are commonplace in mathematics (like groups acting on sets), so that mathematicians can understand what these things are.

Reply:
I don't even know what you mean by "acts additively". I've just added your comment and kept mine, because maybe for someone with your background "acts additively" has some kind of meaning?